bin/pca-import

```diff
@@ -105,7 +105,8 @@ def output_assessment_template():
     assessment["team_lead"] = "Test Testerton"
     assessment["start_time"] = "2017-01-01"
     assessment["end_time"] = "2017-01-21"
-    print(util.to_json(assessment))
+    print(util.to_json(assessment,sort_keys=False)) # Override default behavior to sort JSON key values
+                                                    # Values should be displayed in the order they were built in. 
```

Comment:
I think this makes it much more readable when skimming.

```suggestion
    # Override default behavior to sort JSON key values
    # Values should be displayed in the order they were built in. 
    print(util.to_json(assessment, sort_keys=False))
```

Reply:
Requested changes be here 114a99e

Reply:
@JCantu248 Please add the space between arguments to `utils.to_json()`.

Reply:
See here efe5896